### DISCUSSION TITLE
Installing Guide for Firefly-iii Importer IIS (Windows Server)

### DISCUSSION THREAD

**Discussion by TheFork001** [firefly-iii/firefly-iii#8422] [General]
Step one: I was in my wwroot/firefly-iii directory, launched cmd, and ran this command `composer create-project firefly-iii/data-importer --no-dev --prefer-dist data-importer 1.4.1` Worked for me to download on windows.

If you plan on using this via the internet as an external import my suggestion, correct me if I'm wrong @JC5. leave FIREFLY_III_ACCESS_TOKEN= blank. Doing that will allow firefly-iii importer to reauthenticate you every time. I only set FIREFLY_III_URL=firelyimporter.mywebsite.com and VANITY_URL=fireflyimporter.mywebsite.com I set this to VERIFY_TLS_SECURITY=true and error log to LOG_LEVEL=critical and it allowed my importer to work properly.

Don't forget to heart and up vote if it helped you.

**Comment by JC5** [comment]
The log_level setting should not make a difference (nothing is logged at that level), but nice work! 😁

**Reply by TheFork001** [reply]
Thank you, I hope my guides help. I like the software and hopefully these guides I put up help windows users and makes your software more popular  in the process :)
